main-serve-pipeline.py

```diff
@@ -58,7 +51,12 @@ def log(self, df: pd.DataFrame):
         asyncio.create_task(self.profile_queue.put(profile))
 
     async def __call__(self, request: Request):
-        return str(self.profile.to_summary())
+        try:
+            # TODO remove after bug is fixed
+            # TODO jamie why are there so many NaNs in the min/max cols.
+            return str(self.profile.to_pandas())
+        except:
+            return []
```

Comment:
@jamie256 I notice that my min/max stats are always `NaN` whenever I check by logging the output of `to_pandas`. Am I doing something wrong?

![image](https://user-images.githubusercontent.com/1233709/191841714-5b317be8-7feb-4f1c-abbd-c4c9a5f34354.png)

Reply:
some of the sketches use NaN as the default / uninitialized value, so I would interpret that as these metrics were not executed on that column, which would be expected if the metric is does not apply to the type of values in that column: e.g. Ints/max won't have values for string or float columns.

Reply:
might be worth taking a closer look though, might be something else going on if the metrics are expected to have values on the column.

Reply:
A lot of these columns were numeric. Shouldn't I expect something like the `annual_inc` to have a min/max?